clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml

```diff
@@ -0,0 +1,61 @@
+{{$HostNetworkMode := DefaultParam .CL2_USE_HOST_NETWORK_PODS false}}
```

Comment:
Please rebase this PR.

The worker is not needed anymore.

Reply:
you mean this worker-deployment.yaml is not required?

Reply:
You no longer need any worker - you simply create and read appropriate CR directly from the measurement.

Reply:
Yes this yaml creates the worker pods which listens/watches for the CR created for each unique pair. Hence we need this yaml file.

Reply:
This is my point - this is not the design we want.
We don't want to have a worker to create those CRs. Those should be created by the measurement itself.

Reply:
Yes workers are not creating CRs. Let me explain at high level what is being done in measurement now :

a) Measurement creates worker pods using the yaml based on ratio number (i.e 1:1 or N:M).
b) Then it forms unique pairs of pods (i.e client-server). 
c) It creates a CR object representing a unique pair. It consists of src , dest pod-ip , protocol etc attributes.
d) Worker pods listen for the respective CR object. So only source , destination pod act on this event.
e) Once worker pods finish their work of executing the testcase , they update the status sub-object with test results.
f) Measurement listens to the same CR object. So it gets notified when status is updated and stores the result locally.
g) When measurement gather() is invoked , it will read the locally stored result and return it in displayable format.